lib/project/results-view.coffee

```diff
@@ -21,13 +21,10 @@ class ResultsView extends ScrollView
       view.addClass('selected')
       if not e.ctrlKey
         if e.originalEvent?.detail is 1
-          @editorPromise = view.confirm(pending: true)
+          if view.hasClass('path') or atom.config.get('find-and-replace.openProjectFindResultsInRightPane')
```

Comment:
Which types of entries have the `path` class? Is this worth extracting into a helper for clarity?